device.js

```diff
@@ -24,6 +24,44 @@ class LoupedeckDevice extends EventEmitter {
 
         return rawDevices.flat()
     }
+    static thisDevice = "center";
+    setDevice(x) {
+        this.thisDevice = x;
+    }
+    getScreen(x,y) {
+        switch(this.thisDevice) {
+            default:
+            case "center":
+                return WHICH_DISPLAY.find(wd=> x <= wd.barrier).screen;
+            case "main":
+                return this.thisDevice;
+        }
+    }
+    getButton(x,y) {
```

Comment:
please add some tests to make sure this method works correctly.

I disagree on returning the value `-1`. It should probably `throw` if the x/y coordinates are invalid for any given reason.